Pass.java

```diff
@@ -71,4 +71,5 @@ public String toString() {
                 getCredits() + "\nJourney Points: " +
                 getJourneyPoints() + "\n********************";
     }
+    public void setJourneyPoints(int points){journeyPoints+=points;}
```

Comment:
This isn't needed since the subclasses should be decrementing the journeyPoints like `Pass` is doing